static/js/marker.js

```diff
@@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
 
 		this.$el.width(400);
 		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
-		this.$el.find(".description").text(this.model.get("description"));
+		this.$el.find(".roadType").text(this.model.get("roadType"));
```

Comment:
I understand the Integers idea. Thought it should be kept as description was saved as text in the DB.

I will first change python to pull Integers from the DB.
Then have 'localize_value' function to do the same only in Marker.js